### ISSUE TITLE
There's still has Timeout Issue with the 2.0.601.3402 and .Net framework 4.6.1,SetMinThreads(300,300)

### ISSUE THREAD

**Issue by qjzh124232** [StackExchange/StackExchange.Redis#1219]
Hi,
 I still got this issue with 2.0.601 and had tried all the various ways to solve the issue,but failed.
![image](https://user-images.githubusercontent.com/9276250/64140785-3b4e7a00-ce38-11e9-9553-8f9f0c393650.png)

1.Redis sentinel model with 3 nodes
2.Create  2 ConnectionMultiplexer objects.Use the master end point as the write connection and one of the slaves end point as the read connection.
3.Use the sync mode when the application read the redis and async mode when writing it.
4.Has set the ThreadPool.SetMinThreads(300,300)
5.Has upgraded all the reference libraries to the latest version by now.

And More:
This redis server is  also using by some services with the JDK 8.0 and Lettuce Redis. When my service report the error ,their services are still work well.
So, please help.

**Comment by qjzh124232** [comment]
May be I found the reason.
I used Lazy<ConnectionMultiplexer> to connect the redis sentine server. And used the sentine connection to get the master point and one slave point to create writing connection and reading connection to get or set redis data.

And now, afert I use Lazy<ConnectionMultiplexer> to connect the redis sentine server, I use Lazy<ConnectionMultiplexer> to connect the redis cluster again. Just create one connection. It seemed solve the problam.

So current version does not suppert sentine model, It can only use to monitor the switch-master command so we can use Lazy<ConnectionMultiplexer>  to reconnect the redis server. Am I right?
